mlflow/java/client/pom.xml

```diff
@@ -152,6 +152,10 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.sonatype.plugins</groupId>
+        <artifactId>nexus-staging-maven-plugin</artifactId>
+      </plugin>
```

Comment:
As described in https://stackoverflow.com/a/10483284/6943581, adding a plugin configuration under `pluginManagement` is not enough.